flink-sql-connector-mongodb/pom.xml

```diff
@@ -60,6 +60,10 @@ under the License.
 							<artifactSet>
 								<includes>
 									<include>*:*</include>
+									<include>org.apache.flink:flink-connector-base</include>
+									<include>org.apache.flink:flink-connector-mongodb</include>
+									<include>org.mongodb:*</include>
+									<include>org.bson:*</include>
```

Comment:
I think these new includes are useless as `<include>*:*</include>` already contains them.

Reply:
Thanks @ruanhang1993 to remind me of this.
The old shaded package does not include `flink-connector-base` is that has provided scope.
I removed the useless includes and it works.
Please help review it again when you have time.

Reply:
Thanks @ruanhang1993 to remind me of this.
The old shaded package does not include `flink-connector-base` is that has provided scope.
I removed the useless includes and it works.
Please help review it again when you have time.